pkg/controller/volume/attachdetach/cache/actual_state_of_world.go

```diff
@@ -677,10 +671,49 @@ func (asw *actualStateOfWorld) GetVolumesToReportAttached() map[types.NodeName][
 	return volumesToReportAttached
 }
 
+func (asw *actualStateOfWorld) GetVolumesToReportAttachedForNode(nodeName types.NodeName) (bool, []v1.AttachedVolume) {
+	asw.Lock()
+	defer asw.Unlock()
+
+	var attachedVolumes []v1.AttachedVolume
```

Comment:
nit: `volumesToReportAttached` seems to be easy to understand same as https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/attachdetach/cache/actual_state_of_world.go#L653
```suggestion
	var volumesToReportAttached []v1.AttachedVolume
```